### ISSUE TITLE
[BUG] Misleading Core Power Tags

### ISSUE THREAD

**Issue by compcon** [massif-press/compcon#1736]
<strong>Submitted At: </strong> 10/19/2021 23:05:16<br><br><strong>Issue: </strong><br> In most views, such as the Licenses and Frames view in the Compendium or in the Mech Hanger, the Core System and Core Active of each mech comes with a pair of tags, one with something like "Protocol," "Quick Action," "Full Action," and so on to describe the Action requirement of the Core Active, and another tag with something like "Mission" or "Encounter" or "Next Turn"

However, that second tag (Which comes on the left of the action tag) doesn't exist in the actual Lancer content, and is somewhat misleading. Because it is there and not explained, things like the Encounter tag or the Mission tag are easy to misinterpret as "You can use this once an Encounter" or "You can use this once a Mission," while simply not being a thing in the PDFs. All currently released Core Powers run on a once per Full Repair schedule anyhow, and the current use of "Mission" to describe Core Powers that happen once when activated, and have no further effect and "Encounter" to describe Core Powers that have a lingering effect for the rest of the scene is misleading. If Encounter tagged powers have an effect for the rest of the scene, wouldn't a Mission tagged power last the rest of the Mission? The categories being Mission and Encounter seem to more naturally describe "how often can you use this?"

Furthermore, this feature was apparently forgotten about in the creation of the Long Rim mechs, as all of those mechs have Core Actives that last for an entire Scene, but are labeled Mission as opposed to Encounter, with the one exception of the Kobold.

There are three obvious potential fixes to this:

The first is to rename the "Mission" tag to something more easily understandable as "this happens once, and you're done" such as "Instant" or "Single Use" or something similar, while keeping the Encounter tag the same, and fixing the mistagged Long Rim content. 

You could alternatively just label all the Core Actives "Mission" and have it describe how often you can use the Core Active, as aside from the new playtest content featuring the Efficient tag, all Core Actives can usually be used once a Mission.

The other solution is to just remove those tags entirely, because they misrepresent how Core Powers work, and they simply don't exist in the actual PDFs.
<br><br><strong>Massif Reference: </strong> <br>

**Comment by perijove** [comment]
Input requested from devs: do the tags referenced here tie into the active mode somehow? If so, another possible solution is to make the tags invisible in order to prevent this confusion but leave the functionality.

**Comment by msprijatelj** [comment]
A "quick fix" could be to prevent the Round/Encounter/Mission tags from being rendered, yes.  We wouldn't want to rip out the underlying data since it may be used for tracking uses in Active Mode.  If I were working on this, I'd definitely prefer to modify the language used to more accurately reflect Lancer's lexicon, and ensure that the tags uniformly represented Duration (adding "Instant" as a descriptor if necessary).

@perijove @lenaleciel  How frequently does this confusion occur, on average? Once a week, several times a day, etc.? Just trying to understand the current impact of the issue.

**Comment by perijove** [comment]
A quick search in #rules-discussion indicated four instances of this confusion in the last month, though I think that's more than typical. (I saw @lenaleciel reply to a lot of them, too...)
My preference would be to hide the tags, rather than change their language, as the core book itself does not use tags to describe the duration of core powers. There are a *lot* of people who just read compcon instead of the book (including anyone who is using long rim mechs without purchasing the GM portion) so keeping the language the same is preferable whenever possible, IMO.

**Comment by lenaleciel** [comment]
It's not the most common issue, but it does happen with some regularity and it just sort of happened more commonly as of late, which was the impetus for me trying to look into if there was anything to be done about it on the comp/con side of things.

**Comment by jarena3** [comment]
I think, then, the best solution (at least in the short term) is to just hide the tags in the UI and not worry about the data. That's not a difficult change to make.

**Comment by msprijatelj** [comment]
I created PR to hide the tags, though I would like to keep this issue open/visible for anyone who would like to work on a more robust fix (it'll probably be me haha).

**Comment by msprijatelj** [comment]
Whoops, this was an accidental close.  Reopened for the reason I mentioned previously.

**Comment by jarena3** [comment]
yeah, good catch. Happened automatically with the issue reference
